mllib-local/src/test/scala/org/apache/spark/ml/util/TestingUtils.scala

```diff
@@ -48,7 +48,7 @@ object TestingUtils {
   /**
    * Private helper function for comparing two values using absolute tolerance.
    */
-  private def AbsoluteErrorComparison(x: Double, y: Double, eps: Double): Boolean = {
+  private[ml] def AbsoluteErrorComparison(x: Double, y: Double, eps: Double): Boolean = {
```

Comment:
I don't see this used in the new code, maybe my search is not working properly in browser